docs/containers.conf.5.md

```diff
@@ -412,6 +412,16 @@ Indicates if Podman is running inside a VM via Podman Machine.
 Podman uses this value to do extra setup around networking from the
 container inside the VM to to host.
 
+**machine_image**=""
+
+Default image path or url used when creating a new VM using `podman machine init`. 
+Overrides machine_image_stream. 
```

Comment:
As a user of this, I would have no idea what you are talking about.  I think you should have one option machine_image, which has two constants "testing", "stable" to indicate the two default images or I can specify a full url.  Having two options makes this way too complicated.

Reply:
Updated